<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureWindow</class>
 <widget class="QDialog" name="FeatureWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Features Management</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Descrição</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCodigo">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>61</y>
     <width>49</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Código:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCodigo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>62</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Descricao">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>49</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDescricao">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>92</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEfetivaAlteracao">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpar">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>11</y>
     <width>254</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnNovo">
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAlterar">
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeletar">
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
